### DISCUSSION TITLE
Copy token issue with new release [Sep 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3920] [Discord Threads]
> 🙋‍♂️ **cristianetiquetas**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### worked for me. could you help raise a github issue with project & browser details?  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### Hello  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### I Already intreract in another openned same issue  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### copy token and copy info not working  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### latest release, docker install  
#  
> 💁 **o1lab**  
#### ya we've a fix for it in develop branch   
####    
#### looks like there is a corner case  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1021224551989911582)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
